packages/main/src/OpenedPopupsRegistry.js

```diff
@@ -0,0 +1,27 @@
+import { isEscape } from "@ui5/webcomponents-base/dist/events/PseudoEvents.js";
+
+let registry = [];
+
+const addOpenedPopup = instance => {
+	if (!registry.includes(instance)) {
+		registry.push(instance);
+	}
+};
+
+const removeOpenedPopup = instance => {
+	registry = registry.filter(el => {
+		return el !== instance;
+	});
+};
+
+const getOpenedPopups = () => {
+	return [...registry];
+};
+
+document.addEventListener("keydown", event => {
```

Comment:
IMO make this a separate function, export it and run it explicitly from outside.

Reply:
Also, how are we sure this Esc is not related to some other element inside the popup?

Reply:
I am not :D Any proposals how we can move forward with that interaction?